src/sentry/models/group.py

```diff
@@ -390,21 +374,7 @@ def get_score(self):
         return type(self).calculate_score(self.times_seen, self.last_seen)
 
     def get_latest_event(self):
-        from sentry.models import Event
-
-        if not hasattr(self, '_latest_event'):
```

Comment:
We dropped this local cache here. Is this intentional ?

Reply:
Hmm, I think this was an oversight, will bring this back